PhilipFunkos/src/products.jsx

```diff
@@ -0,0 +1,61 @@
+import "./products.css";
+export default function product() {
+  return (
+    <div className="products-page disable">
+      <div className="product-header">Rare Collectable Funkos</div>
+      <div className="product-card">
```

Comment:
Use an array of data to generate your product cards instead of duplicating layout.